package.json

```diff
@@ -22,6 +22,8 @@
     "docsify": "bin/docsify"
   },
   "scripts": {
+    "lint": "eslint .",
+    "lint:fix": "eslint . --fix",
     "test": "ava e2e/index.js",
```

Comment:
Add the `lint` to `pretest`
also , if possible and if its not conflicting add the `lint` to pre-commit hooks